frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditEmail/EditEmail.tsx

```diff
@@ -129,15 +129,6 @@ export const EditEmail = ({ field }: EditEmailProps): JSX.Element => {
   )
 
   const { data: form } = useCreateTabForm()
-  const isPdfResponseEnabled = useMemo(
-    () => form?.responseMode !== FormResponseMode.Encrypt,
```

Comment:
Hm. For consistency I feel that we can update this to apply for MRF. Looked like it wasn't correctly filtering out MRF previously.